rules/windows/defense_evasion_volume_shadow_copy_deletion_via_powershell.toml

```diff
@@ -0,0 +1,58 @@
+[metadata]
+creation_date = "2021/07/19"
+maturity = "production"
+updated_date = "2021/07/19"
+
+[rule]
+author = ["Austin Songer"]
+description = """
+Identifies use of WMI's Win32_ShadowCopy class or CIM cmdlets for shadow copy deletion on endpoints. This commonly occurs in tandem with ransomware or
+other destructive attacks.
```

Comment:
Suggested description: `Identifies the use of the Win32_ShadowCopy class and related cmdlets to achieve shadow copy deletion. This commonly occurs in tandem with ransomware or other destructive attacks.`